externe_connecties/iv3_definitie_connectie.py

```diff
@@ -10,12 +10,18 @@ def geef_gebruikersvriendelijke_foutmeldingen(fouten):
     """
     Haal gebruikers vriendelijke vertalingen uit fouten.json voor fouten.
     """
-    vriendelijke_fouten = []
-    vertalingen, fouten_bij_ophalen = ophalen_bestand_van_web(IV3_REPO_PATH, 'fouten.json', 'foutenbestand')
-    fouten_overzicht = vertalingen.get("fouten_overzicht")
+    inhoud, foutmeldingen = ophalen_bestand_van_web(IV3_REPO_PATH, 'fouten.json', 'foutenbestand')
+    if foutmeldingen:
+        return foutmeldingen
+
+    fouten_overzicht = inhoud.get("fouten_overzicht")
 
-    # loop over fouten komende uit de controle uitgevoerd door OSF.
     for fout in fouten:
+        if type(fout) == str:
```

Comment:
alle OSF-fouten komen toch als json binnen. waarom dan str-afhandeling?

Reply:
jep, dit is lelijk, ik kan het zo maken dat dit niet nodig is

Reply:
het was omdat er nu 2 typen gebruikersmeldingen zijn, 1 afkomstig van OSF (PF, SF, DF en al die fouten) en 1 van onze eigen code, die gewoon als string kunnen verschijnen. Die van OSF zitten in een dictionary, maar ik zal het splitsen zodat die laatste niet in deze functie worden meegegeven.